lib/Settings/Admin.php

```diff
@@ -35,8 +26,8 @@ public function getForm(): TemplateResponse {
 		$overrideLinkClick = $this->config->getAppValue(Application::APP_ID, 'override_link_click', '0') === '1';
 
 		$adminConfig = [
-			'client_id' => $clientID,
-			'client_secret' => $clientSecret,
+			'client_id' => $clientID === '' ? '' : 'dummy-client-id',
+			'client_secret' => $clientSecret === '' ? '' : 'dummy-client-secret',
```

Comment:
This change is not necessary, admins should be able to see which application is connected.

Reply:
The inputs are type "password" so they won't see the value anyway.